services/slam/builtin/builtin_test.go

```diff
@@ -671,7 +671,7 @@ func TestCartographerDataProcess(t *testing.T) {
 		}()
 
 		cancelCtx, cancelFunc := context.WithCancel(context.Background())
-		c := make(chan int)
+		c := make(chan int, 100)
```

Comment:
Increasing the channel buffer size so that we don't block writing to the channel in the test. It's possible the data process will write to the channel a few more times before the channel is closed.